community/events.page.tsx

```diff
@@ -1239,6 +1239,28 @@ const events = [
     image: require("../static/img/events/commons-france.png"),
     end_date: "October 14, 2025",
   },
+  {
+    name: "EasyA x Flare Harvard Hackathon",
+    description:
+      "This is your chance to meet the Flare Networks team and network with some of the brightest minds in blockchain.",
```

Comment:
This marketing copy come across as a little sensationalist to me. IMO it would be a little better as:

```suggestion
      "Meet the Flare Networks team and network with some of the brightest minds in blockchain.",
```

Reply:
bringing up feedback to copy writers will update you on their response @mDuo13